Meta/CMake/lagom_compile_options.cmake

```diff
@@ -3,6 +3,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/common_compile_options.cmake)
 add_compile_options(-Wno-implicit-const-int-float-conversion)
 add_compile_options(-Wno-literal-suffix)
 add_compile_options(-Wno-maybe-uninitialized)
+add_compile_options(-Wno-shorten-64-to-32)
```

Comment:
This very much belongs into a separate commit

Reply:
Fair enough, moved to a separate commit just about the warning.